integrations/tendermint/src/runtime.ts

```diff
@@ -184,6 +184,19 @@ export default class Tendermint implements IRuntime {
                 );
               }
 
+              // set attribute "ibccallbackerror-error" in ibc event "ibccallbackerror-fungible_token_packet" to empty string
+              if (event.type === 'ibccallbackerror-fungible_token_packet') {
```

Comment:
Does this cause problems for more chains than Osmosis? If not, we should check the `chain_id` before removing.

Reply:
the value is a normal stringified error message like this `amount must be strictly positive: got {827191823872}: invalid token amount`. There is no added value in this if the amount they specify there differs from each node anyway. Since this is an IBC event this could also be an issue on other chains, therefore removing it for all chains is preferred